pkg/cloud/services/network/subnets_test.go

```diff
@@ -2625,6 +2575,183 @@ func TestReconcileSubnets(t *testing.T) {
 					}, nil).AnyTimes()
 			},
 		},
+		{
+			name: "Managed VPC, existing public and private subnets, 2 subnets in spec, custom tags in spec should be created",
```

Comment:
is "Managed VPC" covers "existing subnets"?

Reply:
In BYO infra / [Configuring the AWSCluster Specification](https://cluster-api-aws.sigs.k8s.io/topics/bring-your-own-aws-infrastructure#configuring-the-awscluster-specification) it states:
> Specifying existing infrastructure for Cluster API to use takes place in the specification for the AWSCluster object. Specifically, you will need to add an entry with the VPC ID and the IDs of all applicable subnets into the network field.

So, should it be "unmanaged VPC" when you specify the VPC ID and subnet IDs?

Reply:
[I tried explaining here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/5175/files#r1815091234). Let me know your thoughts, please!